dspace/config/spring/api/openaire-integration.xml

```diff
@@ -0,0 +1,226 @@
+    </bean>
+    <bean id="openaire.author.name" class="org.dspace.importer.external.metadatamapping.MetadataFieldConfig">
+        <constructor-arg value="dc.contributor.author"/>
+    </bean>
+
+    <util:map id="openairePrefixToNamespaceMapping" map-class="java.util.HashMap"
+        key-type="java.lang.String" value-type="java.lang.String">
+        <entry key="http://www.driver-repository.eu/namespace/dri" value="dri" />
+        <entry key="http://namespace.openaire.eu/oaf" value="oaf" />
+        <entry key="http://www.w3.org/2001/XMLSchema-instance" value="xsi" />
+    </util:map>
+    
+    
+    <util:map id="openaireProjectsMetadataFieldMap" key-type="org.dspace.importer.external.metadatamapping.MetadataFieldConfig"
+              value-type="org.dspace.importer.external.metadatamapping.contributor.MetadataContributor">
+        <description>Defines which metadatum is mapped on which metadatum. Note that while the key must be unique it
+            only matters here for postprocessing of the value. The mapped MetadatumContributor has full control over
+            what metadatafield is generated.
+        </description>
+        <entry key-ref="openaireFunding.title" value-ref="openaireFundingTitle"/>
+        <entry key-ref="openaireFunding.acronym" value-ref="openaireFundingAcronym"/>
+        <entry key-ref="openaireFunding.code" value-ref="openaireFundingCode"/>
+        <entry key-ref="openaireFunding.startDate" value-ref="openaireFundingStartDate"/>
+        <entry key-ref="openaireFunding.endDate" value-ref="openaireFundingEndDate"/>
+        <entry key-ref="openaireFunding.funder" value-ref="openaireFundingFunder"/>
+        <entry key-ref="openaireFunding.callidentifier" value-ref="openaireFundingCall"/>
+        <entry key-ref="openaireFunding.summary" value-ref="openaireFundingSummary"/>
+        <entry key-ref="openaireFunding.oamandate" value-ref="openaireFundingOAMandate"/>
+    </util:map>
+
+    <bean id="openaireFundingTitle" class="org.dspace.importer.external.metadatamapping.contributor.SimpleXpathMetadatumContributor">
+        <property name="field" ref="openaireFunding.title"/>
+        <property name="query" value=".//title"/>
+        <property name="prefixToNamespaceMapping" ref="prefixToNamespaceMapping"/>
+    </bean>
+    <bean id="openaireFunding.title" class="org.dspace.importer.external.metadatamapping.MetadataFieldConfig">
+        <constructor-arg value="dc.title"/>
+    </bean>
+
+    <bean id="openaireFundingAcronym" class="org.dspace.importer.external.metadatamapping.contributor.SimpleXpathMetadatumContributor">
+        <property name="field" ref="openaireFunding.acronym"/>
+        <property name="query" value=".//acronym"/>
+        <property name="prefixToNamespaceMapping" ref="prefixToNamespaceMapping"/>
+    </bean>
+    <bean id="openaireFunding.acronym" class="org.dspace.importer.external.metadatamapping.MetadataFieldConfig">
+        <constructor-arg value="oairecerif.acronym"/>
```

Comment:
These `oairecerif.*` metadata fields don't exist in DSpace.  Should these be either removed or commented out?  It looks like lines 141-211 might all have this problem.  I don't think these will work in DSpace unless I'm missing something?